src/components/ResultCount.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <Text text="{{ count }} movie(s) found" />

```

Comment:
`text="{{ count }} movie(s) found"`, this syntax is not working
please add computed properties to component script, like this
```
{
   ...
   computed: {
      countText() {
          return `${this.count} movie(s) found`
      }
   }
   ...
}
```
and use it in template 
`<Text :text="countText" />`

Reply:
@rnesteruk13 fixed